src/main/java/com/ani/backend/service/pdfService.java

```diff
@@ -0,0 +1,277 @@
+package com.ani.backend.service;
+import java.io.IOException;
+import org.springframework.stereotype.Service;
+import com.lowagie.text.Document;
+import com.lowagie.text.Element;
+import com.lowagie.text.Font;
+import com.lowagie.text.FontFactory;
+import com.lowagie.text.Image;
+import com.lowagie.text.PageSize;
+import com.lowagie.text.Paragraph;
+import com.lowagie.text.Phrase;
+import com.lowagie.text.Rectangle;
+import com.lowagie.text.Table;
+import com.lowagie.text.pdf.CMYKColor;
+import com.lowagie.text.pdf.PdfPCell;
+import com.lowagie.text.pdf.PdfPTable;
+import com.lowagie.text.pdf.PdfWriter;
+import jakarta.servlet.http.HttpServletResponse;
+
+@Service
+public class pdfService {
+
+      // adding A4-size-page
+        public void export(HttpServletResponse response) throws IOException{
+        float width = PageSize.A4.getWidth() + 60; // Increase the width by 10 units
+        float height = PageSize.A4.getHeight() + 30; // Increase the height by 10 units
+        Rectangle customSize = new Rectangle(width, height);
+        Document document = new Document(customSize);
+        PdfWriter.getInstance(document, response.getOutputStream());
+        document.setPageSize(customSize);
```

Comment:
you have already set the size of the Document in Document document = new Document(customSize); No need of these lines